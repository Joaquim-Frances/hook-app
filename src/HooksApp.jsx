import React from 'react'

export const HooksApp = () => {
    return (
        <div>
            <h1>Hola Mon!</h1>
        </div>
    )
}
