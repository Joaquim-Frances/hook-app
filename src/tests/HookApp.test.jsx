import { shallow } from "Enzyme"
import { HooksApp } from "../HooksApp";
import '@testing-library/jest-dom';
import React from 'react';

describe('Testing HookApp', () => {
    
    test('should match the snapshot', () => {

        const wrapper = shallow(<HooksApp />);

        expect(wrapper).toMatchSnapshot();
        
    })
    
    
})
