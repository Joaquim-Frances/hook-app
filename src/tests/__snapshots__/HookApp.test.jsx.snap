// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HookApp should match the snapshot 1`] = `
<div>
  <h1>
    Hola Mon!
  </h1>
</div>
`;
